components/LandingPageComponents/GiveSupportSection.js

```diff
@@ -84,7 +84,7 @@ const GiveSupportSection = () => (
     <HeroSectionHeader>
       <h2>How you can support others</h2>
       <HeroSectionButtonContainer>
-        <Button type='secondary' size='large' shape='round' block>See all Activities</Button>
+        <Button type='secondary' href='/a/offer' size='large' shape='round' block>See all Activities</Button>
```

Comment:
This should be wrapped in <Link rather than have an href in the button. 

also text is not translated

Reply:
oh okay i'll wrap it in Link. 

@avowkind so i didn't make that change because that is another ticket . I want to follow the ticket and when I have done this i'll make the change. But if you want me to make all the changes on this ticket even though it not in this ticket then I can do that. But I would need to change that ticket and ad it  as a subtask.

Reply:
@avowkind  by wraping it in <Link> would that make it translated ??